documentation/modules/deploying/proc-deploy-kafka-mirror-maker.adoc

```diff
@@ -21,6 +22,9 @@ In this procedure, we use the following example files:
 * `examples/mirror-maker/kafka-mirror-maker.yaml`
 * `examples/mirror-maker/kafka-mirror-maker-2.yaml`
 
+IMPORTANT: If deploying MirrorMaker 2 clusters to run in parallel, each instance must use unique names for internal Kafka Connect topics. 
+To do this, xref:con-kafka-connect-mm2-multiple-instances-{context}[configure each MirrorMaker 2 instance to replace the defaults].  
```

Comment:
As explain in the other comment. We should be more clear when does it need to be used with MM2.